### ISSUE TITLE
Noise In the Output

### ISSUE THREAD

**Issue by zanubpro** [lastdefiance20/TryYours-Virtual-Try-On#11]
![image](https://github.com/lastdefiance20/TryYours-Virtual-Try-On/assets/138089628/c8250afa-18c7-4647-9e22-869b6e5c8f22)
![image](https://github.com/lastdefiance20/TryYours-Virtual-Try-On/assets/138089628/866d8e48-e491-40c1-9252-e1ed454e9a0e)

Hi @lastdefiance20 there seems to be lot of noise in the output and full sleeves and even sleeveless t-shirts often gets their sleeves short? what could be the issue? Is it that dependencies may be the problems. Why is it giving half sleeves all the time?
Is there any parameter which i can tweak to get better results?
Though this is the issue while running locally.

**Comment by nazapip** [comment]
@lastdefiance20 .. Any idea?. i even tried with many versions of pytorch and torchvision but no improvements.

**Comment by blt232018** [comment]
> ![image](https://user-images.githubusercontent.com/138089628/249669096-c8250afa-18c7-4647-9e22-869b6e5c8f22.png) ![image](https://user-images.githubusercontent.com/138089628/249669164-866d8e48-e491-40c1-9252-e1ed454e9a0e.png)
> 
> Hi @lastdefiance20 there seems to be lot of noise in the output and full sleeves and even sleeveless t-shirts often gets their sleeves short? what could be the issue? Is it that dependencies may be the problems. Why is it giving half sleeves all the time? Is there any parameter which i can tweak to get better results? Though this is the issue while running locally.

have you ever tried to put this cloth on other model? I mean you may try to modify the data in test_pair.txt, then see what happens.

**Comment by nazapip** [comment]
> > ![image](https://user-images.githubusercontent.com/138089628/249669096-c8250afa-18c7-4647-9e22-869b6e5c8f22.png) ![image](https://user-images.githubusercontent.com/138089628/249669164-866d8e48-e491-40c1-9252-e1ed454e9a0e.png)
> > Hi @lastdefiance20 there seems to be lot of noise in the output and full sleeves and even sleeveless t-shirts often gets their sleeves short? what could be the issue? Is it that dependencies may be the problems. Why is it giving half sleeves all the time? Is there any parameter which i can tweak to get better results? Though this is the issue while running locally.
> 
> have you ever tried to put this cloth on other model? I mean you may try to modify the data in test_pair.txt, then see what happens.

Yes, i have tried it on other models bur luckily i got this issue resolved of halsf sleeves but still not able  generate a proper image like still there is a skin color problem in the generated image
![image](https://github.com/lastdefiance20/TryYours-Virtual-Try-On/assets/137103045/b152899f-a600-43ae-8f08-62a6bfcccd3a)

**Comment by codeforeveryone05** [comment]
How to run this project locally. .Please guide us
